stdlib/_typeshed/__init__.pyi

```diff
@@ -26,6 +26,14 @@ Self = TypeVar("Self")  # noqa Y001
 class IdentityFunction(Protocol):
     def __call__(self, __x: _T) -> _T: ...
 
+# stable
+class SupportsNext(Protocol[_T_co]):
+    def __next__(self) -> _T_co: ...
+
+# stable
+class SupportsANext(Protocol[_T_co]):
```

Comment:
Nitty: Maybe this should be `SupportsAnext`?

We have `SupportsAclose` elsewhere.
Anext is the camel case equivalent of `anext` / maybe reads more distinguishably from `SupportsNext`? Don't feel strongly though!

Reply:
For what it's worth 3.10rc2 renamed PyAiter_Check to PyAIter_Check in the C API (https://docs.python.org/3.10/whatsnew/changelog.html#id2).